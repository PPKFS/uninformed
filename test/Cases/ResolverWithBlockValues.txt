Home is a room.

To announce (msg - text) to (target - a person):
	say "[target]: [msg]".

To vigorously announce (msg - text) to (target - a person):
	say "Hey ";
	announce msg to target.

Joe is a person in home.

When play begins:
	vigorously announce "Where you going with that gun in your hand?" to Joe.
