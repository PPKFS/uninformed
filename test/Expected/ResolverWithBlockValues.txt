56 words
¶home¶ ¶is¶ ¶a¶ ¶room¶ ¶.¶ ¶|__¶ ¶to¶ ¶announce¶ ¶(¶ ¶msg¶ ¶-¶ ¶text¶ ¶)¶ ¶target¶ ¶person¶ ¶:¶ ¶say¶ ¶"[target]: [msg]"¶ ¶vigorously¶ ¶"Hey "¶ ¶;¶ ¶joe¶ ¶in¶ ¶when¶ ¶play¶ ¶begins¶ ¶"Where you going with that gun in your hand?"¶ 
27 distinct words
¶Home¶ ¶home¶ 0a
¶is¶ ¶is¶ 20
¶a¶ ¶a¶ 20
¶room¶ ¶room¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶To¶ ¶to¶ 20
¶announce¶ ¶announce¶ 20
¶(¶ ¶(¶ 20
¶msg¶ ¶msg¶ 20
¶-¶ ¶-¶ 20
¶text¶ ¶text¶ 20
¶)¶ ¶)¶ 20
¶to¶ ¶to¶ 20
¶(¶ ¶(¶ 20
¶target¶ ¶target¶ 20
¶-¶ ¶-¶ 20
¶a¶ ¶a¶ 20
¶person¶ ¶person¶ 20
¶)¶ ¶)¶ 20
¶:¶ ¶:¶ 20
¶say¶ ¶say¶ 41
¶"[target]: [msg]"¶ ¶"[target]: [msg]"¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶To¶ ¶to¶ 20
¶vigorously¶ ¶vigorously¶ 20
¶announce¶ ¶announce¶ 20
¶(¶ ¶(¶ 20
¶msg¶ ¶msg¶ 20
¶-¶ ¶-¶ 20
¶text¶ ¶text¶ 20
¶)¶ ¶)¶ 20
¶to¶ ¶to¶ 20
¶(¶ ¶(¶ 20
¶target¶ ¶target¶ 20
¶-¶ ¶-¶ 20
¶a¶ ¶a¶ 20
¶person¶ ¶person¶ 20
¶)¶ ¶)¶ 20
¶:¶ ¶:¶ 20
¶say¶ ¶say¶ 41
¶"Hey "¶ ¶"Hey "¶ 20
¶;¶ ¶;¶ 20
¶announce¶ ¶announce¶ 41
¶msg¶ ¶msg¶ 20
¶to¶ ¶to¶ 20
¶target¶ ¶target¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Joe¶ ¶joe¶ 20
¶is¶ ¶is¶ 20
¶a¶ ¶a¶ 20
¶person¶ ¶person¶ 20
¶in¶ ¶in¶ 20
¶home¶ ¶home¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶When¶ ¶when¶ 20
¶play¶ ¶play¶ 20
¶begins¶ ¶begins¶ 20
¶:¶ ¶:¶ 20
¶vigorously¶ ¶vigorously¶ 41
¶announce¶ ¶announce¶ 20
¶"Where you going with that gun in your hand?"¶ ¶"Where you going with that gun in your hand?"¶ 20
¶to¶ ¶to¶ 20
¶Joe¶ ¶joe¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
